src/Radio Buttons/main.vala

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4
```

Comment:
```suggestion
#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1
```

Reply:
I suggested it for the sake of consistency with other demos, but I guess it's fine just gtk4

Reply:
Let's go for _only when needed_, like in this demo. Should I add it to the STYLEGUIDE.md?